indy-besu/vdr/src/client/client.rs

```diff
@@ -30,27 +33,40 @@ impl LedgerClient {
     ///  client to use for building and sending transactions
     pub fn new(
         chain_id: u64,
-        node_address: &str,
+        mut node_addresses: Vec<&str>,
         contract_configs: &[ContractConfig],
     ) -> VdrResult<LedgerClient> {
+        let rpc_node_address = node_addresses.remove(0);
```

Comment:
If we still need to pass RPC node it will be better to have such API:
```
    pub fn new(
        chain_id: u64,
        rpc_node: &str,
        contracts: &[ContractConfig],
        quorum_config: Option<&QuorumConfig>,
    ) -> VdrResult<LedgerClient> {
```
where QuorumConfig
```
pub struct QuorumConfig {
    nodes: Vec<&str>,
    request_retries: u8,
    request_timeout: u8,
}
```
and ledger client will store quorum handle if config passed and valid 
request quorum if ihandler was initialized

Reply:
Changed API